frontend/package.json

```diff
@@ -10,13 +10,19 @@
   },
   "dependencies": {
     "expo": "^35.0.0",
+    "expo-constants": "^8.0.0",
```

Comment:
maybe it's too new for the expo sdk.
I got this
```
Some of your project's dependencies are not compatible with currently installed expo package version:
 - expo-constants - expected version range: ~7.0.0 - actual version installed: ^8.0.0
Your project may not work correctly until you install the correct versions of the packages. To install the correct versions of these packages, please run: expo install [package-name ...]
```

Reply:
thx Kyya